server/seeders/20181010134458-demo-article.js

```diff
@@ -1,9 +1,29 @@
 module.exports = {
   up: queryInterface => queryInterface.bulkInsert('Articles', [{
-    title: 'title of article',
-    body: 'body of article',
-    slug: 'title-of-article',
-    description: 'description of article',
+    title: 'title1',
```

Comment:
I changed this so I could search for similar articles of different authors in my test